$NetBSD: patch-aa,v 1.2 2006/10/05 09:32:02 obache Exp $

Set up cbrowser so that it can find its subsidiary tcl files

--- cbrowser.orig	2005-07-23 18:02:24.000000000 +0200
+++ cbrowser
@@ -2989,7 +2989,7 @@
 while {![catch {file readlink $link} result]} {
   set link [file join [file dirname $link] $result]
 }
-set sourcedir [file dirname $link]
+set sourcedir [file dirname [file dirname $link]]/share/cbrowser;
 # Set up the auto_path to find things in the directory where this lives.
 lappend auto_path $sourcedir
 
